packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -131,6 +131,8 @@
       </div>
       <!-- Right content -->
       <div class="dt-message-input__bottom-section-right">
+        <slot name="smsCount" />
```

Comment:
Add a comment like this one:
`<!-- @slot Slot for sms count -->`
so that the slot will show in the docs